src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -995,7 +996,10 @@ private void RefreshPackageContents()
                 .ToObservableCollection();
 
             var items = new Dictionary<string, PackageItemRootViewModel>();
-
+            if (!String.IsNullOrEmpty(this.CurrentPackageDirectory))
+            {
+                var v = 1;
```

Comment:
huh?

Reply:
uhh, sorry..remains of my debugging trip, will remove